web/bulk_rendering/renderer_core.ts

```diff
@@ -187,7 +187,7 @@ namespace com.keyman.renderer {
     }
 
     setActiveDummy() {
-      com.keyman['DOMEventHandlers'].states.activeElement = BatchRenderer.dummy;
+      com.keyman['dom']['DOMEventHandlers'].states.activeElement = BatchRenderer.dummy;
```

Comment:
There were a few odds and ends left over from the previous PR needed to fix `bulk_rendering`.